deploy/crds/kubevirt_v1alpha3_nodemaintenance_crd.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: nodemaintenances.kubevirt.io
+spec:
+  group: kubevirt.io
+  names:
+    kind: NodeMaintenance
+    listKind: NodeMaintenanceList
+    plural: nodemaintenances
+    singular: nodemaintenance
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          type: object
+        spec:
+          type: object
+        status:
+          type: object
```

Comment:
Should we be a bit more specific here? Or is this enough?

Reply:
Actually this was generated by the operator-sdk,  I think for now its enough but ill give it a second look.